typings/index.d.ts

```diff
@@ -30,20 +30,26 @@ export class GiveawaysManager<ExtraData = any> extends EventEmitter {
     public options: GiveawaysManagerOptions<ExtraData>;
     public ready: boolean;
 
+    protected generateMainEmbed(giveaway: Giveaway<ExtraData>, lastChanceEnabled?: boolean): EmbedBuilder;
+    protected generateEndEmbed(giveaway: Giveaway<ExtraData>, winners: GuildMember[]): EmbedBuilder;
+    protected generateNoValidParticipantsEndEmbed(giveaway: Giveaway<ExtraData>): EmbedBuilder;
```

Comment:
are we really considering them as protected? ik cause of the extend stuff, but it can also be used to just get the embeds for comparisions/anything else

Reply:
It wasn't present earlier and iirc it's marked as private in jsdoc so I made it protected so that people get strict type when extending. Will make it public then.